models/articles.model.js

```diff
@@ -1,27 +1,45 @@
 const db = require("../db/connection.js");
-const {
-  validateArticleId,
-  validateIncVotes,
-} = require("../utils/utils.validation.js");
+const { validateIncVotes } = require("../utils/utils.validation.js");
 const { articleExists } = require("./utils.model.js");
 
-exports.fetchArticles = () => {
-  const queryStr = `
+exports.fetchArticles = (sort_by = "created_at", order = "desc") => {
+  const validSortColumns = [
+    "article_id",
+    "title",
+    "author",
+    "body",
+    "topic",
+    "created_at",
+    "votes",
+    "comment_count",
+  ];
+  const validOrder = ["asc", "desc", "ASC", "DESC"];
```

Comment:
Is there something we could do to the string beforehand to make sure it's uppercase?

Reply:
Good point! I did it further away in the code, but it makes sense to do it before this line. :) Thank you! Will change this. :)